sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sample/PathogenTest.java

```diff
@@ -64,9 +68,21 @@
     @Column(nullable = false)
     private PathogenTestResultType testResult;
 
+    @Column(nullable = true)
+    private Boolean testResultVerified;
+
+    @Column(length=512, nullable = true)
+    private String testResultText;
+
     @DatabaseField(dataType = DataType.DATE_LONG, canBeNull = false)
     private Date testDateTime;
 
+    @DatabaseField(foreign = true, foreignAutoRefresh = true, maxForeignAutoRefreshLevel = 3, canBeNull = false)
```

Comment:
Don't use not null constraints (apart from those unfortunately already there) in the app because SQLite doesn't support removing those at a later point in time when they might become obsolete